__tests__/app.test.js

```diff
@@ -193,31 +193,126 @@ describe("GET /api/articles/:article_id/comments", () => {
         });
       });
   });
-  test("should respond with a status code:400 when passed an invalid article_id type ", () => {
+  test('should respond with a status code:400 when passed an invalid article_id type ', () => {
     return request(app)
-      .get("/api/articles/one/comments")
+      .get('/api/articles/one/comments')
       .expect(400)
       .then(({ body }) => {
-        expect(body.msg).toBe("Invalid ID Type");
+        expect(body.msg).toBe('Invalid ID Type');
       });
   });
 
-  test("should respond with a status: 404 when passed a valid but non-existent id", () => {
+  test('should respond with a status: 404 when passed a valid but non-existent id', () => {
     return request(app)
-      .get("/api/articles/200/comments")
+      .get('/api/articles/200/comments')
       .expect(404)
       .then(({ body }) => {
-        expect(body.msg).toBe("Article Not Found");
+        expect(body.msg).toBe('Article Not Found');
       });
   });
 
-  test("should respond with a status: 200 and an empty array when passed a valid and existing id that has no comments", () => {
+  test('should respond with a status: 200 and an empty array when passed a valid and existing id that has no comments', () => {
     return request(app)
-      .get("/api/articles/2/comments")
+      .get('/api/articles/2/comments')
       .expect(200)
       .then(({ body }) => {
         expect(body).toEqual(expect.any(Array));
         expect(body).toHaveLength(0);
       });
   });
 });
+
+describe('POST /api/articles/:article_id/comments', () => {
+  test('should respond with a status code:201 and a returned comment object, with the fields: comment_id, body, author, article_id, created_at.', () => {
+    const date1 = new Date(Date.now());
+    const date1InSeconds = Math.floor(date1.getTime());
+
+    const postableComment = {
+      username: 'icellusedkars',
+      body: 'comment for test',
+    };
+
+    return request(app)
+      .post('/api/articles/2/comments')
+      .send(postableComment)
+      .expect(201)
+      .then(
+        ({
+          body: {
+            returnComment: [body],
```

Comment:
I'd like it if you renamed this second body on line 242 please - it's more readable, but also more trustable because we can be sure which body we are referring to on the following lines